src/lsst/cm/tools/core/panda_utils.py

```diff
@@ -1,12 +1,34 @@
-from typing import Any
+from typing import Any, TextIO
 
+import yaml
 from pandaclient import Client, panda_api
 
 from lsst.cm.tools.core.db_interface import DbInterface, JobBase
 from lsst.cm.tools.core.slurm_utils import SlurmChecker
 from lsst.cm.tools.core.utils import StatusEnum
 
 
+def print_errors_aggregate(stream: TextIO, errors_aggregate: dict[int, dict[str, Any]]) -> None:
+    copy_dict = {}
+    for key, val in errors_aggregate.items():
+        if not isinstance(val, list):
+            copy_dict[key] = val
+            continue
+        else:
```

Comment:
Minor readability suggestion: either lose the `else` and outdent `copy_list = []`. or indent the following bits into the `else`?  (Otherwise there's a visual distinction between `copy_list = []` and the rest, but logically they all hang together.)